From eedfa57953d2d6255d53cb098c3f81cbde9187f7 Mon Sep 17 00:00:00 2001
From: Cristian Stoica <cristian.stoica@nxp.com>
Date: Mon, 22 Feb 2016 11:47:27 +0200
Subject: [PATCH 1/2] Disable installing header file provided by another
 package

(original patch Signed-off-by: Denys Dmytriyenko <denys@ti.com>)
Signed-off-by: Cristian Stoica <cristian.stoica@nxp.com>

Upstream-Status: Inappropriate [ OE specific ]


--- a/Makefileold	2017-04-18 14:54:40.588438842 +0800
+++ b/Makefile	2017-04-18 14:55:40.308436774 +0800
@@ -33,7 +33,6 @@
 
 modules_install:
 	$(MAKE) $(KERNEL_MAKE_OPTS) modules_install
-	install -m 644 -D crypto/cryptodev.h $(DESTDIR)/$(includedir)/crypto/cryptodev.h
 
 clean:
 	$(MAKE) $(KERNEL_MAKE_OPTS) clean
